### PR TITLE
Fedramp machineconfig

### PR DESCRIPTION
**Pull request by theautoroboto**
Adding configuration to write chrony.conf for fedramp specific time servers

### CODE REVIEW

**Comment by dustman9000** [comment]
Verified config.yaml exists only for FR clusters.
/hold cancel

File: deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:,server%20129.6.15.28%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Aserver%20132.163.97.1%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Aserver%20132.163.96.1%20prefer%20iburst%20minpoll%204%20maxpoll%204%0Adriftfile%20%2Fvar%2Flib%2Fchrony%2Fdrift%0Artcsync%0Alogdir%20%2Fvar%2Flog%2Fchrony
+          mode: 420
+          overwrite: true
+          path: /etc/chrony.conf
+  kernelArguments:
+    - loglevel=7
```
Comment (gsleeman) [inline deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml:39]: what are the kernelArguments for?

Comment (theautoroboto) [reply deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml:39]: Very good point.  These should not be in there. updating config now

File: deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+apiVersion: machineconfiguration.openshift.io/v1
```
Comment (dustman9000) [inline deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml:1]: Can you separate out these yaml files into their respective manifests? 00-fedramp-chrony-worker.yaml and 00-fedramp-chrony-master.yaml

**Comment by dustman9000** [comment]
looks good
/lgtm
/approve

**Pull request merged**